src/pages/hall-of-fame.jsx

```diff
@@ -2,12 +2,14 @@ import ScoreBoard from "@/components/home/score-board";
 import { useFetchPastLeaderboardsQuery } from "@/store/api";
 
 const HallOfFame = () => {
+  const useFetchData = (params) => useFetchPastLeaderboardsQuery({ ...params, year: 2024 });
+
   return (
     <ScoreBoard
       pageTitle={"Hall of Fame | Bashaway"}
-      title={"Hall of Fame 2023"}
+      title={"Hall of Fame 2024"}
       subTitle={" A tribute to legendary warriors who once marched amongst us with unwavering valour"}
-      useFetchData={useFetchPastLeaderboardsQuery}
```

Comment:
useFetchPastLeaderboardsQuery is imported but removed here and replaced with useFetchData. Any reason? @icy-r